lib/fast_excel/binding.rb

```diff
@@ -13,7 +13,11 @@ module Libxlsxwriter
     "libxlsxwriter.so"
   end
 
-  ffi_lib File.expand_path("./../../../libxlsxwriter/lib/#{LIB_FILENAME}", __FILE__)
+  if Gem.loaded_specs['fast_excel']
+    ffi_lib File.join(Gem.loaded_specs['fast_excel'].extension_dir, LIB_FILENAME)
+  else
+    ffi_lib File.expand_path("../../#{LIB_FILENAME}", __FILE__)
```

Comment:
I don't know if there is more elegant solution. It's in order to make tests and examples pass, because during those execution, `Gem.loaded_specs['fast_excel']` is nil. That's why I also make this https://github.com/Paxa/fast_excel/pull/91/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R36

Reply:
There might be other circumstances where this would fail, so maybe this code would be better:
```ruby
begin
  ffi_lib File.join(Gem.loaded_specs['fast_excel'].extension_dir, LIB_FILENAME)
rescue LoadError
  ffi_lib File.expand_path("../../#{LIB_FILENAME}", __FILE__)
end
```

Speaking of tests, make sure to add 3.2 to the rubies in ".github/workflows/test.yml". That way you'll find any other issues that the test suite would smoke out.

Reply:
Also, I think that the reason `Gem.loaded_specs['fast_excel']` is nil is because the Gemfile is missing the `gemspec` declaration. Also, gems kinda shouldn't have Gemfile.lock declared in their tracked files, especially when testing against multiple ruby versions.

In Gemfile, change this code:
```ruby
gem 'ffi'
# gem 'ffi_gen', require: false
```
to this:
```ruby
gemspec
```

Then in ".github/workflows/test.yml" change this code:
```yaml
ruby: [2.6, 2.7, '3.0', 3.1]
```
to this:
```yaml
ruby: [2.6, 2.7, '3.0', 3.1, 3,2]
```

In ".gitignore" add this line:
```
Gemfile.lock
```

Then run this command:
```sh
git rm Gemfile.lock
```